modules/protobuf/27.0/presubmit.yml

```diff
@@ -0,0 +1,67 @@
+matrix:
+  platform: ["debian11", "macos", "ubuntu2004"]
+  bazel: [6.x, 7.x]
+
+tasks:
+  verify_targets:
+    name: "Verify build targets"
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+    - '--host_cxxopt=-std=c++14'
+    - '--cxxopt=-std=c++14'
+    - '--host_copt=-Wno-deprecated-declarations'
```

Comment:
Ideally we shouldn't fail on warnings treated as errors here at all instead of having to chase these down each time.

Reply:
my hope was that this specific one, coming from abseil https://buildkite.com/bazel/bcr-presubmit/builds/5808#018fc07d-8fb3-46e7-85c4-575697a4524c would be fixed by updating abseil so we wouldn't need to apply this upstream and the next real release would just have the fix. IIUC the reason it fails is only because when building with the bazel host configuration we're getting -Werror, it wouldn't fail if we were only building for the target configuration